addons/pt/saft/extensions.go

```diff
@@ -166,6 +245,73 @@ var extensions = []*cbc.Definition{
 			i18n.EN: "Tax exemption reason code",
 			i18n.PT: "Código do motivo de isenção de imposto",
 		},
+		Desc: i18n.String{
+			i18n.EN: here.Doc(`
+				AT's ~TaxExemptionCode~ (Código do motivo de isenção de imposto) is a code that
+				specifies the reason the VAT tax is exempt in a Portuguese invoice. When the ~exempt~ tag
+				is used, one of the following must be defined in the ~ext~ map's ~pt-exemption-code~
+				property:
+
+				| Code  | Description                                                                                              |
```

Comment:
So listing the table again is probably redundant all of these will be presented in the output, indeed, they already are: https://docs.gobl.org/addons/pt-saft-v1#pt-saft-exemption

The exception to this rule is when there is a specific mapping using special keys, and the link is not obvious.